mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompiler.java

```diff
@@ -158,211 +159,215 @@ public abstract CompletableFuture<List<AutoImportsResult>> autoImports(String na
 	public abstract CompletableFuture<List<TextEdit>> insertInferredType(OffsetParams params);
 
 	/**
-	 * Return the text edits for inlining a value.
```

Comment:
oh, I think metals github action runs automatically before push (I had to apply to be able to push). Maybe you can try [difftastic](https://github.com/Wilfred/difftastic) or ignore [whitespaces in PR](https://stackoverflow.com/questions/37007300/how-to-ignore-whitespace-in-github-when-comparing)

Reply:
It's fine, but I think the formatting was done by the editor actually, but maybe we never run any formatting on it before :thinking: Would probably be cleaner to have that as a separate commit because otherwise any blame in the file will point at you change :sweat_smile: